frontend/public/components/_quota.scss

```diff
@@ -5,3 +5,65 @@
 .co-resource-quota-empty {
   font-size: 15px;
 }
+
+.co-resource-quota-chart-row {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  margin-bottom: 20px;
+  .co-resource-quota-chart {
+    min-width: 150px;
+  }
+}
+
+.co-resource-quota-scope__label {
+  font-weight: 500;
+}
+.co-resource-quota-scope__description {
+  font-size: 12px;
+  font-weight: 300;
+}
+
+.quota-dashboard-row {
+  display: flex;
+  .group {
+    height: 100%;
+    .group__title {
+      border-left: none;
+      border-right: none;
+    }
+    // condensed charts, scope label & desc on same line
+    .group__graphs {
+      .co-resource-quota-chart-row {
+        margin-bottom: 2px;
+      }
+      .graph-wrapper {
+        margin-top: 2px;
+        margin-bottom: 0px;
+        padding-top: 10px;
+      }
+      dl {
+        padding-left: 24px;
+        padding-right: 24px;
+        dd {
+          margin-bottom: 2px;
+          line-height: 1.2;
+          .co-resource-quota-scope__label {
+            display: inline;
+            font-size: 12px;
+          }
+          .co-resource-quota-scope__description {
+            display: inline;
+            padding-left: 6px;
+          }
+        }
+      }
+    }
+  }
+}
+
+.quota-dashboard-row > [class*='col-'] {
```

Comment:
I'm wary of overriding Bootstrap's defaults.  I recommend either use Bootstrap straight or roll your own layout.